client/my-sites/site-settings/disconnect-site/style.scss

```diff
@@ -17,6 +17,60 @@
 	}
 }
 
+.disconnect-site__down-flow {
+	> p {
+		font-size: 14px;
+		text-align: center;
+		color: var( --color-neutral-50 );
+	}
+}
+
+.disconnect-site__actions {
+	$title-size: 16px;
+
+	margin-bottom: 1.5em;
+
+	.card {
+		// .card.is-card-link’s padding-right
+		padding-left: 48px;
+
+		.formatted-header {
+			margin-top: 0;
+			margin-bottom: 0;
+		}
+
+		.formatted-header__title {
+			font-size: $title-size;
+		}
+
+		.formatted-header__subtitle {
+			margin-bottom: 0;
+		}
+	}
+
+	.disconnect-site__action-icon {
+		position: absolute;
+
+		$padding: 16px; // .card’s padding
+		$icon-height: 24px; // Gridicon’s intrinsic height
+		$text-height: $title-size * 1.2; // formatted-header__title’s line-height (without em)
+		top: $padding - ( $icon-height - $text-height ) / 2;
+		left: $padding;
+	}
+
+	.disconnect-site__confirm-icon {
+		color: var( --color-primary );
```

Comment:
Thanks for bringing it up. Testing it just now I feel `--studio-blue-40` is the best one, since it brings all the icons to the same level of brightness.

Reply:
done in dc3aaefb6e062f94e37bfbfe8668dfac049eecff (i’ve used `--color-primary-40` based on the [colour docs](https://wpcalypso.wordpress.com/devdocs/docs/color.md#semantic-colors) guidance, but let me know if that’s no good)